src/benchmarklib/file_based_table_generator.hpp

```diff
@@ -13,8 +14,17 @@ class FileBasedTableGenerator : virtual public AbstractTableGenerator {
 
   std::unordered_map<std::string, BenchmarkTableInfo> generate() override;
 
+  /**
+   * @param add_constraints_callback is called after the table data has been added to the StorageManager.
```

Comment:
Maybe add "... is called (via `_add_constraints()`) after ..."

Reply:
```cpp
  /**
   * Set @param add_constraints_callback to define table constraints, if available. It is called by _add_constraints
   * after tables have been generated.
   */
```

Better?

Reply:
Perfect.